packages/sit-onyx/src/components/OnyxSelectInput/OnyxSelectInput.vue

```diff
@@ -207,9 +220,22 @@ useAutofocus(input, props);
             tabindex="-1"
             :disabled="disabled || props.readonly || props.loading"
           >
-            <OnyxIcon :icon="iconChevronDownUp" />
+            <slot name="icon">
+              <OnyxIcon :icon="iconChevronDownUp" />
+            </slot>
+          </button>
+          <button
```

Comment:
Always including the clear button for the select will break some current components :/ This might even be a breaking change to include it by default. Personally, I'd prefer to implement it in a separate standalone PR instead of putting it in here since its not really related to the timepicker itself.

This would currently break:
- version select in playground
- grid playground
- select pagination

If its already a breaking change for us, it will most likely also be one for all users.